apollo-router/src/compute_job/mod.rs

```diff
@@ -140,25 +176,42 @@ where
             }
         }
     });
+
     let queue = queue();
-    queue.send(priority, wrapped_job).map_err(|e| match e {
-        SendError::QueueIsFull => {
-            u64_counter!(
-                "apollo.router.compute_jobs.queue_is_full",
-                "Number of requests rejected because the queue for compute jobs is full",
-                1u64
-            );
-            ComputeBackPressureError
-        }
-        SendError::Disconnected => {
-            // This never panics because this channel can never be disconnect:
-            // the receiver is owned by `queue` which we can access here:
-            let _proof_of_life: &'static AgeingPriorityQueue<_> = queue;
-            unreachable!("compute thread pool queue is disconnected")
-        }
-    })?;
+    let job = Job {
+        ty: compute_job_type,
+        job_fn: wrapped_job_fn,
+        queue_start: Instant::now(),
+    };
+
+    queue
+        .send(Priority::from(compute_job_type), job)
+        .map_err(|e| match e {
+            SendError::QueueIsFull => {
+                u64_counter!(
+                    "apollo.router.compute_jobs.queue_is_full",
+                    "Number of requests rejected because the queue for compute jobs is full",
+                    1u64
+                );
```

Comment:
```suggestion
```
I don't think this is needed anymore. `apollo.router.compute_jobs.duration` has a status of RejectedQueueFull that we can get this value from.

Reply:
Hmm, OK we'll have to deave it but maybe comment that it should be removed in the next major version.